expr/literals.go

```diff
@@ -7,13 +7,13 @@ package expr
 import (
 	"bytes"
 	"fmt"
+	"google.golang.org/protobuf/types/known/anypb"
```

Comment:
why did this import move? Can we please put it back in the import list with the rest of the imports that are not stdlib imports?

Reply:
as elsewhere, -1 on review comments around import order. If we want a specific order, add as precommit tests.

Reply:
yes, it should be fixed by pre-commit. I will add a pre-commit action to detect imports in future PR